final_task/rss_reader/databaseEmulation.py

```diff
@@ -0,0 +1,94 @@
+import pickle
+import logging as log
+import os
+import datetime
+
+from typing import Any
+
+
+now = datetime.datetime.now
+
+
+class Singleton(type):
+    _instances = {}
+
+    def __call__(cls, *args, **kwargs):
+        if cls not in cls._instances:
+            cls._instances[cls] = super(Singleton, cls).__call__(
+                *args,
+                **kwargs
+            )
+        return cls._instances[cls]
```

Comment:
It is better to not use such thing as singleton in this app. It is not needed at all.